test/e2e/manifests/csi-nfs-controller.yaml

```diff
@@ -0,0 +1,108 @@
+---
```

Comment:
There should also be a CSIDriver object with attachRequired: false

Reply:
I didn't add that manifest as it is already present here - https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/deploy/kubernetes/csi-nfs-driverinfo.yaml
We're reusing this under `make kind-install-driver`
The newly added node and controller manifest in this PR use a driver image built locally in the CI workflow

Reply:
Ah I see. Maybe instead of copying the entire manifest to a new folder and only changing the image name, can we use something like kustomize to patch the image name and tag?